spawn_worker/build.rs

```diff
@@ -0,0 +1,11 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present Datadog, Inc.
+
+fn main() {
```

Comment:
I have trouble figuring out the life cycle of the trampoline. Can you help me out ? 
Compiled -> Stored somewhere in library? (this is the place I'm missing) -> written to memory --> Exec

Reply:
https://github.com/DataDog/libdatadog/pull/110/files#diff-29a0a855c8fb3650574054b856d1145f82bb1b8c8ccd5bc50720a2d8b9e76424R4-R5
Its included as bytes by rustc